src/resdk/tables/variant.py

```diff
@@ -379,15 +359,20 @@ def _parse_file(self, file_obj, sample_id, data_type) -> pd.Series:
             s = sample_data["Base_T"]
 
         s.name = sample_id
+
+        # It si common for mutations_table.tsv to contains duplicate variants
+        # For now, eep the first one and drop the rest of them.
+        s = s[~s.index.duplicated(keep="first")]
+
         return s
 
-    def postprocess_variants_table(self, df, data_type):
+    def _postprocess_variants_table(self, df, data_type):
```

Comment:
Because of the change in `mutations-table` mentioned in the previous comment, I think we can remove this function. There will not be any positions reported for which we know that do not contain variant by default. Furthermore, if two variants are present in the collection for the same position but different nucleotide change, we would lose one of them if we discarded "fake variant".

Reply:
<3